### ISSUE TITLE
Question: modeling provenance information without knowing whether an actor is the 'owner' or 'custodian'?

### ISSUE THREAD

**Issue by sdevalk** [linked-art/linked.art#473]
Hi all,

A question about the modeling of provenance information:

In my dataset - with metadata of artworks - there is quite some provenance information about the current and previous owners and custodians of artworks. I could use LA’s Provenance pattern for this: [https://linked.art/model/object/ownership/](https://linked.art/model/object/ownership/)

There is one catch, though: I don’t know whether a piece of provenance information is about the ‘owner’ or the ‘custodian’ — all I know is that an artwork was 'transferred' to someone else. So I cannot say that some actor is the `current_owner` or `current_custodian`. Similarly, I cannot say that an artwork `changed_ownership_through` an `Acquisition` — my data doesn’t say that there was an acquisition (it could be some other ‘transfer’ activity).

Is there a recommendation to model provenance information in a generic fashion, without having to classify the actors involved as ‘owners’ or ‘custodians’?

Thanks!

**Comment by azaroth42** [comment]
I started typing that there isn't a way to do that, but then I realized there is ... it's just really terrible.

TL;DR:  There isn't a good way to do it.

For masochists and @sdevalk:

First, read [this](https://linked.art/model/provenance/rights/#rights-and-their-acquisition)

Okay... so we established a way to transfer a Right of ownership for things that aren't HumanMadeObjects (such as transfer of the right to perform time based media or whatever). If the knowledge is only that there was either the right of custody or the right of ownership, but unsure which, then there could be a transfer of a less specific right that encompasses both. And then one imagines some other party lost that same Right, otherwise it's not a transfer.
(So read [this](https://linked.art/model/provenance/rights/#transfer-of-shares))

Meaning we could do...

```
{
      "type": "RightAcquisition",
      "establishes": {
        "type": "Right",
        "_label": "Some Indistinct Property Right Over the Object",
        "classified_as": [
          {
            "id": "http://local-thesaurus/right",
            "type": "Type",
            "_label": "Indistinct Property Right"
          }
        ],
        "possessed_by": [
          {
            "type": "Group",
            "_label": "Transferred-to-party"
          }
        ],
        "applies_to": [
          {
            "type": "HumanMadeObject",
            "_label": "Painting"
          }
        ]
      },
      "invalidates": {
        "type": "Right",
        "_label": "Previous Indistinct Property Right Over the Object",
        "classified_as": [
          {
            "id": "http://local-thesaurus/right",
            "type": "Type",
            "_label": "Indistinct Property Right"
          }
        ],
        "possessed_by": [
          {
            "type": "Group",
            "_label": "Transferring-party"
          }
        ],
        "applies_to": [
          {
            "type": "HumanMadeObject",
            "_label": "Painting"
          }
        ]
      }
    }
```

Like I said ... it's not great.

**Comment by azaroth42** [comment]
Other use case:  unclear if the object is consigned, owned or simply loaned.  (Getty PI, Frick)
Rijks also.

**Comment by azaroth42** [comment]
Proposal: Flesh out a generic Transfer class that sits between Activity and Transfer of Custody / Acquisition. Check with further use cases, hopefully from Getty Provenance Index and Lynn Rother. 
Once there's something with evidence, then raise an issue with the SIG, without expectation it will go to base any time soon.

**Comment by sdevalk** [comment]
Hi @azaroth42 Thank you very much! I wasn't aware of the 'Transfer of Rights' pattern - thanks for pointing me to it. But,  ouch - it's indeed rather painful (and I'm not that much of a masochist to enjoy it).

I'd welcome a proposal for a generic Transfer class!

**Comment by cbutcosk** [comment]
Meant to ping this when this came up in conversation the other day--we definitely hit a need for a `Transfer` class on [JDCRP](https://pilot-demo.jdcrp.org). The solution @azaroth42 outlines in the first item in #479 seems sensible as a profile alignment. I might add aliasing `transferred_title_to` -> `transferred_to` and `transferred_custody_to` -> `transferred_to`, etc in the scoped contexts for `Acquisition` and `TransferOfCustody`, which would make working in the nodes in JSON-LD a little easier as well.

Some of the other practical issues we encountered:
- Sibling `Acquisition`, `TransferOfCustody`, and `Move` lacking a useful superclass makes it harder than it should be to interface with ledgers, logbooks, and inventories because `Activity` has a very coarse picture of its parts. Provenance "routes" lose a fair amount of semantic resolution when they appear on summary reification nodes (eg, the `Activity` of the transfers between two waypoints, through the lifecycle of an object until its suspected or known destruction, etc).
- Changing event type creates large and hard-to-reason-about patches: all the activity's actor props move to new props (eg, `transferred_title_to` -> `transferred_custody_to`) and so on.
- Prov events can sometimes be prematurely turned into one of the existing classes when the factual universe is (usefully and in-scope) ambiguous. Often a researcher will only know "this truck driver took it to Place A" or "it was checked in here" but whose _custody_ it is in at those points is ambiguous above a certain noise floor.

Anyway in the call notes upstream seems less interested in changes (maybe? tough to tell), but a `Transfer` superclass that usefully bridges `TransferOfCustody`, `Acquisition`, and `Move` (and `Payment`--hadn't realized that but makes sense) would be very nice.

**Comment by azaroth42** [comment]
Ontology / Context level is done. Next for documentation in model.

**Comment by azaroth42** [comment]
Done in #652
